packages/select-react/src/Select.tsx

```diff
@@ -132,9 +134,16 @@ export function Select({
 
     const [elementRef] = useAnimatedHeight(dropdownIsShown);
 
+    function getWidthAsStyle(width?: string): CSSProperties | undefined {
+        if (width) {
+            return { width };
+        }
+        return undefined;
+    }
+
     return (
-        <div data-testid="jkl-select" className={componentClassName}>
-            <Label variant={variant} forceCompact={forceCompact}>
+        <div data-testid="jkl-select" className={componentClassName} style={getWidthAsStyle(width)}>
```

Comment:
Dette vil lage trøbbel for BM @joms ?

Reply:
> Dette vil lage trøbbel for BM?

Hmm, det hadde jeg ikke tenkt på.. Er godt mulig man i så fall fortsatt bør sette standard `width` i stilarket fremfor som default prop. Inline-stilen vil vel uansett få presedens om man setter den.

Reply:
Det hadde vært digg om det er mulig å unngå inline styling

Reply:
Kan evt. la være å sette `width` i det hele tatt (bare `min-width`), så kan man sette det i en klasse hvis man vil. Problemet er at det må settes på et element inne i `Select`, så er nok fortsatt bedre å bruke `width`-propen. Men da har dere en en slags workaround tilgjengelig for BM i hvert fall 😅

Reply:
@joms, jeg merger inn denne. Nå legges det i hvert fall ikke til noen inline styles der det ikke var det før, så får dere si fra om det blir noen utfordringer i bruk senere 😊